### ISSUE TITLE
Topic pattern

### ISSUE THREAD

**Issue by arnitolog** [papirosko/mm2-lag#2]
@papirosko is it possible to configure topic patterns instead of explicit topic names? 
I have a use case when topics can be created dynamically on the source cluster and mirrored to the remote cluster. And it is impossible to specify the names because nobody knows what the topic name will be (we have a topic per tenant for some products). But it is possible to describe all of them with some pattern.

**Comment by papirosko** [comment]
not possible currently.

2 things to implement:
- select topics based on pattern
- reload kafka topics periodically and subscribe to newly created ones

**Comment by papirosko** [comment]
@arnitolog please try `papirosko/mm2-lag:patterns`
configuration should be like this:
```
traffic:
  - from: source1
    to: target
    connectorName: mm2-msc-source1
    mm2OffsetsTopic: "mm2-kafka-connect-offsets"
    # you can use both `topics` and `topicsPatterns` together
    topics:
      - another_topic
    topicsPatterns: # use regular java regex syntax
      - topic_name.*
```

**Comment by papirosko** [comment]
merged to main, released in `0.3.0`

**Comment by arnitolog** [comment]
yeah, that works for me. Thanks a lot, @papirosko for the amazing tool
